src/build/get_build_parameters.mjs

```diff
@@ -14,7 +14,7 @@
 
 import minimist from 'minimist';
 
-const VALID_PLATFORMS = ['linux', 'windows', 'ios', 'macos', 'android', 'browser'];
+const VALID_PLATFORMS = ['linux', 'windows', 'ios', 'macos', 'android', 'browser', 'storybook'];
```

Comment:
Why does storybook need to be a separate platform? It seems like the wrong abstraction.

Reply:
Also, it doesn't seem like that platform is used

Reply:
Ah oops I added this then decided against it. Forgot to remove.